app/src/components/market/common/market_scale/index.tsx

```diff
@@ -530,6 +541,27 @@ export const MarketScale: React.FC<Props> = (props: Props) => {
     },
   ]
 
+  const bondedValueBoxData = [
+    {
+      title: `${currentPredictionNumber.toFixed(2)} ${unit}`,
+      subtitle: 'Predicted Outcome',
+    },
+    {
+      title:
+        currentAnswerBond && collateral
+          ? `${formatBigNumber(currentAnswerBond, collateral.decimals)}  ${collateral && collateral.symbol}`
+          : '-',
+      subtitle: 'Bonded',
+    },
+    {
+      title:
+        currentAnswer && collateral
+          ? `${formatBigNumber(new BigNumber(currentAnswer), collateral.decimals)} ${unit}`
+          : '-',
+      subtitle: isBonded ? 'Pending Outcome' : 'Final Outcome',
+    },
```

Comment:
For some reason, when I was testing, after setting an outcome the UI didn't update to reflect the pending outcome, even after refreshing and setting a new outcome. Can you reproduce? Code looks good so maybe a subgraph issue?

Reply:
https://user-images.githubusercontent.com/33226956/113284780-6df97680-92ea-11eb-95da-70e9a6d57f2b.mov

Hmmm could be. For me it works. I will leave it unresolved if maybe @pimato sees something